ros2bag/ros2bag/verb/record.py

```diff
@@ -281,6 +302,8 @@ def validate_parsed_arguments(args, uri) -> str:
     if args.compression_queue_size < 0:
         return print_error('Compression queue size must be at least 0.')
 
+    return ''
```

Comment:
should return None for success?

```suggestion
    return None
```